src/wizards/newspack/views/settings/seo/verification-codes.tsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * Components for managing SEO verification codes.
+ */
+
+/**
+ * WordPress dependencies.
+ */
+import { __ } from '@wordpress/i18n';
+import { Fragment } from '@wordpress/element';
+import { ExternalLink } from '@wordpress/components';
+
+/**
+ * Internal dependencies.
+ */
+import { Grid, TextControl } from '../../../../../components/src';
+
+function VerificationCodes( {
+	data,
+	setData,
+}: {
+	data: SeoData[ 'verification' ];
+	setData: ( v: SeoData[ 'verification' ] ) => void;
+} ) {
+	return (
+		<Grid>
+			<TextControl
+				label="Google"
+				onChange={ ( google: string ) => setData( { ...data, google } ) }
+				value={ data.google }
+				help={
+					<Fragment>
+						{ __( 'Get your verification code in', 'newspack' ) + ' ' }
+						<ExternalLink href="https://www.google.com/webmasters/verification/verification?tid=alternate">
```

Comment:
Looks like the URL for this settings page has changed. It's now `https://search.google.com/search-console/ownership?resource_id=<url>` where `<url>` is the encoded URL for the site, e.g.

`https://search.google.com/search-console/ownership?resource_id=https%3A%2F%2Fnewspack.com%2F`